lib/src/install/baseline.rs

```diff
@@ -256,6 +256,16 @@ pub(crate) fn install_create_rootfs(
             "BIOS-BOOT",
             Some("21686148-6449-6E6F-744E-656564454649"),
         );
+    } else if cfg!(target_arch = "powerpc64") {
+        // BIOS-BOOT
+        partno += 1;
+        sgdisk_partition(
+            &mut sgdisk.cmd,
+            partno,
+            "0:+4M",
+            "BIOS-BOOT",
```

Comment:
I wonder if we should try to match https://github.com/coreos/coreos-assembler/blob/38235218542c39a29255933d3cea64e91754e85f/src/create_disk.sh#L206 and make the label `PowerPC-PReP-boot`?

Reply:
Yes, good point. I have pushed the modified code.